CHANGES.rst

```diff
@@ -25,6 +25,11 @@
 Changes
 =======
 
+Version 1.0.0b4 (released 2017-08-09)
+-------------------------------------
+
+- Fixes for Content Security Poclicy (CSP).
+
```

Comment:
There is no need to keep the previous version here. You can take a look at other release commits [like this one](https://github.com/inveniosoftware/invenio-assets/commit/0e5fe263d359d06151854a0b93b005a031fbd59a). As long as we are in alpha/beta we just replace the previous message.